files/es/web/html/attributes/index.md

```diff
@@ -25,7 +25,7 @@ El atributo IDL, tambien conocido como propiedad JavaScript. Estos son atributos
 
 La mayoria del tiempo, los atributos IDL pueden retornar sus valores como realmente son usados. Por ejemplo, el `type` por defecto para los elementos {{HTMLElement("input")}} es "texto", entonces si defines `input.type="foobar",` el elemento `<input>` sera de tipo texto (en apariencia y comportamiento) pero el valor del Atributo de contenido "type" sera "foobat". De cualquier manera, el `type` del atributo IDL retornara el string "text".
 
-Los atributos IDL no son siempre strings; por ejemplo, `input.maxlength` es un numero (marcado como long). Cuando se usan atributos IDL, lees o defines valores del tipo deseado, entonces `input.maxlength `siempre retornara un numero y cuando definas `input,maxlength`, esperara un numero. Si se le entrega otro tipo, se convierte antomaticamente a numero como se especifica en las reglas del estandar de JavaScript para conversiones de tipo.
```

Comment:
Unforturnately, you need to review all the files after running the markdownlint, because things like this well happen. There space was misplaced, and requires manually adjusting, because Markdownling will just correctly remove the inner space, but you're still left with a word stuck to the code span. Not sure if Prettier might fix some of those, but it's not entirely integrated upstream

Reply:
Thank you so much, I put this as draft due to I still pending to have the manual review of each rendered file, but I'll take a note on this cases

Reply:
Hi! @nschonni I remember you did a lot of this corrections in past markdown cleanups, all of these were done manually?